fms/models/gpt_bigcode.py

```diff
@@ -299,11 +295,7 @@ def forward(
         x: torch.LongTensor,
         mask: Optional[torch.Tensor] = None,
         position_ids: Optional[torch.LongTensor] = None,
-        past_key_value_states: Optional[
-            Tuple[
-                torch.FloatTensor,
-            ]
-        ] = None,
+        past_key_value_states: Optional[Tuple[torch.FloatTensor,]] = None,
```

Comment:
Should this just be torch.Tensor instead of torch.FloatTensor? Based on changes for mypy